decentraland/explorer
# fix: Correct handling of WebGL compatibility errors

# What does this PR fix? 
## Fixes https://github.com/decentraland/unity-renderer/pull/720
We should check the WebGL compatibility of the current browser and show our "not supported" error screen if it is not supported:

![image](https://user-images.githubusercontent.com/64659061/124774687-4a277b80-df3e-11eb-8a61-fac69aa9a52f.png)

## How to test it?
1. Open Firefox and type `about:config` in the url bar.
2. Search the flag `webgl.disabled` and turn it to `false`.
3. Go to https://play.decentraland.zone/branch/fix/Handle-the-WebGL-support/index.html?ENV=org
4. Notice that the "not supported" error screen is showed.
5. Go to the `webgl.disabled` again and turn it to `true` now.
6. Go to https://play.decentraland.zone/branch/fix/Handle-the-WebGL-support/index.html?ENV=org again
7. Notice that the world is loaded correctly.
